packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -112,8 +113,8 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         throwOnBadResponse: true,
         fetchOptions
     }
-    const apiClients = useMemo(() => {
-        return {
+    const apiClients: ApiClients = useMemo(() => {
```

Comment:
All the changes are in commerce-sdk-react. So the assumption is for customers to consume it, they'll need to upgrade their commerce-sdk-react dependency to the latest version.